build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
shouldn't this be in the `android` folder? (and the other `kts` file too)

Reply:
No. it's a special high-level ROOT project configuration that allows working on library code without switching between sample and lib. fast development cycle in mind.

Reply:
ok.. we should make it so that the library can be developed _through_ the example app. Most repos in react-native-community do that, it requires a few steps, see https://github.com/react-native-community/react-native-google-signin/search?q=edited+for+example+app&unscoped_q=edited+for+example+app

or use https://github.com/mweststrate/relative-deps